cmake/linker_script/common/kobject-priv-stacks.cmake

```diff
@@ -0,0 +1,43 @@
+#Keep in synch with include/zephyr/linker/kobject-priv-stacks.ld
+if(CONFIG_USERSPACE)
+  if(CONFIG_GEN_PRIV_STACKS)
+    #/* Padding is needed to preserve kobject addresses
+	  #* if we have reserved more space than needed.
+	  #*/
```

Comment:
Weird comment style, remove the C multi-line comment and only keep `#` comments. And fix indentation.